addons/account/tests/__init__.py

```diff
@@ -15,8 +15,6 @@
 from . import test_account_analytic
 from . import test_account_payment
 from . import test_account_bank_statement
-from . import test_account_move_partner_count
```

Comment:
Moved the test in another file, as it was a whole setUpClass without any difference, for a single test.
Did it for 2 of these and some in expense too